feature-group-definitions/async-await.yml

```diff
@@ -1,4 +1,5 @@
 name: Async functions
+description: The `async` and `await` keywords allow you to use the asynchronous, promise-based behavior of a function without using promise chains.
```

Comment:
As I noted before it's possible to pick holes in the "using promise chains" bit here, but I think this is close enough to the motivation in practice.